content/blog/quick-kafka-installation/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Quick Kafka Installation on Windows"
```

Comment:
You can add title as `Setting Up and Running Apache Kafka on Windows OS`.

Reply:
Done